models/PostModel.js

```diff
@@ -156,10 +166,20 @@ const getReplies = `
     ON p."postId" = r."parentPostId"
   INNER JOIN app_user AS u
     ON p."userId" = u."userId"
+
   WHERE p."parentPostId" = $2
+    AND p."deleted" = FALSE
   ORDER BY "numberOfLikes" DESC, "timestamp" DESC;
 `;
 
+const deletePost = `
+  UPDATE post
+  SET deleted = TRUE
+  WHERE "postId" = $1
+  AND "userId" = $2
+  AND deleted = FALSE
+  RETURNING "postId", deleted;`;
```

Comment:
Do we need to return these?

Reply:
We don't actually need them. I think I was planning to use that response in the frontend but I chose to optimistically update the post list through the reducer only.